tck/src/main/java/org/eclipse/configjsr/converters/implicit/SomeEnumToConvert.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.configjsr.converters.implicit;
+
+/**
+ * Sample enum to test implicit Converter Rules
+ * @author <a href="mailto:struberg@yahoo.de">Mark Struberg</a>
+ */
+public enum SomeEnumToConvert {
```

Comment:
It looks you are not testing the Enum converter in `ImplicitConverterTest`, right?

Reply:
yikes, right, will add it - txs!